Task_customFilter.js

```diff
@@ -0,0 +1,29 @@
+
+  
+  
+  // my custom array filter
+  // write your own method which implements Array.prototype.filter
+  
+  function customFilter(callback) {
+    let array = [];
+    for(let i = 0; i < this.length; i++) {
+        if(this[i] >= 1000) {
```

Comment:
The `customFilter` should rely on callback result!
It should work with any kind of passed callback, for example: 
```
  let badSalary = arraySalary.customFilter((value, i, arr) => {
      return value < 300;

  });